astrovascpy/scipy_petsc_conversions.py

```diff
@@ -231,7 +235,7 @@ def coomatrix2PETScMat(L):
     return A
 
 
-def coomatrix2PETScMat_naive(L):
+def coomatrix2PETScMat_naive(L):  # pragma: no cover
```

Comment:
Can't you add a small test instead?

Reply:
The two functions that I marked as `pragma no cover` are not used in the library.  In an old comment, Christos told me to keep them for possible future references.  Actually, I don't think we will ever use them, so we could remove them.  What do you think?

Reply:
If you are pretty sure these functions won't be used, you can remove them. There will remain in the git history anyway.